generators/app/templates/src/redux/auth/actions.js

```diff
@@ -0,0 +1,67 @@
+import { NavigationActions } from 'react-navigation';
+
+import * as AuthService from '../../services/AuthService';
+import { stringArrayToObject } from '../../utils/reduxUtils';
+
+export const actions = stringArrayToObject(
+  ['LOGIN', 'LOGIN_SUCCESS', 'LOGIN_FAILURE', 'LOGOUT', 'AUTH_INIT'],
```

Comment:
🤔  Should these action names be sorted by name ? What do you think ? @sbalay @IgnacioAbadie @ignaciosantise